cmd/osbuild-composer/config.go

```diff
@@ -51,17 +53,35 @@ func (c *ComposerConfigFile) weldrDistrosImageTypeDenyList() map[string][]string
 	return distrosImageTypeDenyList
 }
 
+// GetDefaultConfig returns the default configuration of osbuild-composer
+// Defaults:
+// - 'ec2' and 'ec2-ha' image types on 'rhel-85' are not exposed via Weldr API
+func GetDefaultConfig() *ComposerConfigFile {
+	return &ComposerConfigFile{
+		WeldrAPI: WeldrAPIConfig{
+			map[string]WeldrDistroConfig{
+				"rhel-85": {
```

Comment:
I wonder if we should make this more generic... `rhel-8*`? maybe even `rhel-*`?

Reply:
Probably won't make much of a difference, but it will be one less thing to keep in mind when we will be adding support for 8.6. Ideally the `ami` should be an alias to `ec2` in the future and we should have a way to optionally compress the image or choose to install RHUI client instead of using RHSM. At that point we won't need this default configuration any more and will be able to expose the image type via all APIs, but we are not there yet. I'll make the change, probably for all RHEL version using `rhel-*`.